activefires_pp/post_processing.py

```diff
@@ -560,19 +574,22 @@ def regional_fires_filtering_and_publishing(self, msg, regional_fmask, afsff_obj
             regions_with_detections = regions_with_detections + 1
             fmda['region_name'] = regional_fmask[region_name]['attributes']['Kod_omr']
 
+            out_filepath = os.path.join(self.output_dir, pout.compose(fmda))
+            logger.debug("Output file path = %s", out_filepath)
             data_in_region = afdata[regional_fmask[region_name]['mask']]
 
-            filepaths = self.create_output(data_in_region, fmda, self.regional_outputs)
-            for filepath in filepaths:
-                if not filepath:
-                    logger.warning("Something wrong happended storing regional " +
-                                   "data to Geojson - area: {name}".format(name=str(region_name)))
-                    continue
+            feature_collection = geojson_feature_collection_from_detections(data_in_region,
+                                                                            platform_name=fmda['platform'])
+            if feature_collection is None:
```

Comment:
I think it would be more pythonic to raise an error from `geojson_feature_collection_from_detections` and handle that

Reply:
Agree, will do!